Tool for bulk transcription

tilmankamp opened the pull request.

reuben commented on transcribe.py:
```diff
@@ -0,0 +1,154 @@
+                bar.update(len(transcripts))
+            bar.finish()
+            transcripts.sort(key=lambda t: t[0])
+            transcripts = list(map(lambda t: {'start': int(t[0]), 'end': int(t[1]), 'transcript': t[2]}, transcripts))
```
How about a comprehension instead?

```python
transcripts = [{'start': int(start), 'end': int(end), 'transcript': transcript} for start, end, transcript in transcripts]
```

reuben commented on transcribe.py:
```diff
@@ -0,0 +1,154 @@
+        with open(src_path, 'r') as catalog_file:
+            catalog_entries = json.load(catalog_file)
+        catalog_entries = list(map(lambda e: (resolve(catalog_dir, e['audio']), resolve(catalog_dir, e['tlog'])),
+                                   catalog_entries))
```
Same here?
```python
catalog_entries = [(resolve(catalog_dir, e['audio']), resolve(catalog_dir, e['tlog'])) for e in catalog_entries]
```

tilmankamp commented on util/audio.py:
```diff
@@ -0,0 +1,146 @@
+            os.remove(self.tmp_file_path)
+
+
+def load_audio(audio_path, audio_format=DEFAULT_FORMAT):
```
Removed it

tilmankamp commented on util/audio.py:
```diff
@@ -0,0 +1,146 @@
+                voiced_frames = []
+        frame_index += 1
+    if triggered:
+        pass
```
Yes - removed it.

tilmankamp commented:
@reuben Could  you have a second look (review)? I addressed all your comments, but there are some important changes in `transcribe.py` to prevent memory leaks...
